### DISCUSSION TITLE
Xcode plugin to compare images

### DISCUSSION THREAD

**Discussion by andriyslyusar** [pointfreeco/swift-snapshot-testing#680] [Ideas]
I am quite new to the project and idea of snapshot testing, but while looking around I come across https://github.com/orta/snapshots and https://github.com/Antondomashnev/FBSnapshotsViewer projects to compare snapshot images in Xcode using plugin or using separate open source application. Both of the projects are not maintained anymore, but the ideas of viewing screenshot in the Xcode is very interesting.

Do we know any other similar projects currently maintained? Would it be nice to build out-of-box integration with this framework?

**Comment by tdrhq** [comment]
We've built this: https://github.com/screenshotbot/screenshotbot-oss

Essentially, we move the screenshot comparisons to an external service. It isn't expected to work with the record/verify flow, but instead it just records in CI and notifies you on code review. 

There's also a similar open source project by Workday: https://github.com/workday/vizzy

**Reply by andriyslyusar** [reply]
This is not exactly the same type of tools.

**Reply by tdrhq** [reply]
@andriyslyusar Oh yeah, definitely not the same as what you're asking. However, they do come with better image comparison tools. Was just giving you options. Hopefully someone else has a better idea.
